scss/_popover.scss

```diff
@@ -179,20 +183,28 @@
 
 // Offset the popover to account for the popover arrow
 .popover-header {
-  padding: var(--#{$prefix}popover-header-padding-y) var(--#{$prefix}popover-header-padding-x);
+  padding: var(--#{$prefix}popover-header-padding-top) var(--#{$prefix}popover-header-padding-x) var(--#{$prefix}popover-header-padding-bottom); // Boosted mod
```

Comment:
Feels weird to use `--#{$prefix}popover-header-padding-top` here instead of `--#{$prefix}popover-header-padding-y`.

In terms of defensive CSS, it could be better to have it the other way around. Right now, if you only have a `.popover-header` without content, it's not displayed correctly. But maybe you already talked about it and it was too complicated to do?

Reply:
> In terms of defensive CSS, it could be better to have it the other way around. Right now, if you only have a `.popover-header` without content, it's not displayed correctly. But maybe you already talked about it and it was too complicated to do?

Just pushed a new version who can work with popovers with header and content, with header but no content and no header but content.

I think it just need a new name for `$popover-header-padding-bottom` but it's currently 8:30PM so... 😄